cmd/traefik/storeconfig.go

```diff
@@ -69,47 +72,69 @@ func runStoreConfig(kv *staert.KvSource, traefikConfiguration *TraefikConfigurat
 		}
 
 		if traefikConfiguration.GlobalConfiguration.ACME != nil {
-			var object cluster.Object
 			if len(traefikConfiguration.GlobalConfiguration.ACME.StorageFile) > 0 {
-				// convert ACME json file to KV store
-				localStore := acme.NewLocalStore(traefikConfiguration.GlobalConfiguration.ACME.StorageFile)
-				object, err = localStore.Load()
-				if err != nil {
-					return err
-				}
-			} else {
-				// Create an empty account to create all the keys into the KV store
-				account := &acme.Account{}
-				err = account.Init()
-				if err != nil {
-					return err
-				}
-
-				object = account
+				return migrateACMEData(traefikConfiguration.GlobalConfiguration.ACME.StorageFile, traefikConfiguration.GlobalConfiguration.ACME.Storage, kv)
 			}
+		}
+		return nil
+	}
+}
 
-			meta := cluster.NewMetadata(object)
-			err = meta.Marshall()
-			if err != nil {
-				return err
-			}
+func migrateACMEData(fileName, storageKey string, kv *staert.KvSource) error {
```

Comment:
could you add some blank lines in this method.

Reply:
done